src/components/Toast/Toast.css

```diff
@@ -0,0 +1,36 @@
+.toast {
+  align-items: center;
+  display: flex;
+  position: fixed;
+  bottom: 24px;
+  right: 24px;
+  background-color: #08d69f;
+  width: auto;
+  height: 56px;
+  border-radius: 5px;
+}
+
+.toast__icon {
+  margin-left: 8px;
+}
+
+.toast__button {
+  position: absolute;
+  transform: translate(-10%, -90%);
```

Comment:
you only need transform translate if you want to center the element. In this case the button doesn't need to be centered